pkg/registry/Makefile.am

```diff
@@ -11,7 +11,9 @@ registry_VIEWS = \
 	pkg/registry/views/imagestream-meta.html \
 	pkg/registry/views/imagestream-page.html \
 	pkg/registry/views/imagestream-panel.html \
-	pkg/registry/views/project-new-dialog.html \
+	pkg/registry/views/project/add-group-dialog.html \
```

Comment:
I think we should keep a flat directory hierarchy here, and name these files like this:

project-new-dialog.html
group-new-dialog.html
user-add-dialog.html